.dir-locals.el

```diff
@@ -81,13 +81,10 @@
   (clojure-indent-style . always-align)
   (cljr-favor-prefix-notation . nil)
   (clojure-docstring-fill-column . 118)
-  (cider-preferred-build-tool . clojure-cli))
-
- ("shared"
-  (clojure-mode
-   (cider-default-cljs-repl . shadow-select)
-   (cider-shadow-default-options . "node-repl")
-   (cider-preferred-build-tool . shadow-cljs)))
+  (cider-preferred-build-tool . clojure-cli)
+  (cider-default-cljs-repl . shadow-select)
+  (cider-shadow-default-options . "node-repl")
+  (cider-preferred-build-tool . shadow-cljs))
```

Comment:
We have had Cljc code in the main `src` and `test` directories for a while now so the CIDER config shouldn't only applied to `shared` anymore